packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -114,39 +135,58 @@ contract DependencyRegistryV0 is
         _transferOwnership(_adminACLContract);
     }
 
+    /**
+     * @notice Adds a new license type that can be used by dependencies.
+     */
+    function addLicenseType(bytes32 _licenseType) external {
+        _onlyAdminACL(this.addLicenseType.selector);
+        // @dev the add function returns false if set already contains value
+        require(_licenseTypes.add(_licenseType), "License type already exists");
+        require(
+            _licenseType != bytes32(""),
+            "License type cannot be empty string"
```

Comment:
nit: typically prefer error messages to be <=31 characters, such that they fit into a single, 32-byte chunk. This helps reduce contract bytecode size (and deployment costs).